internal/seclang/parser.go

```diff
@@ -57,13 +62,13 @@ func (p *Parser) FromFile(profilePath string) error {
 		file, err := fs.ReadFile(p.root, profilePath)
 		if err != nil {
```

Comment:
For example here, if the file could not be read, the currentDir set in line 61 would remain to that failing value.